### DISCUSSION TITLE
Time recording could not be stopped

### DISCUSSION THREAD

**Discussion by P1xelC0da** [kimai/kimai#4027] [Questions & Answers]
```
There was a problem saving.
Maximum 8:00 hours allowed.
```

So I went to System -> Settings -> Calendar -> Start of normal business hours & End of normal business hours and modified them to 00:00 and 23:59 but it still doesn't work. How can I modify this to allow me to work 23-24 hours?

**Comment by kevinpapst** [comment]
System -> Settings -> Timesheet.
There is a minute setting for the max duration.

**Reply by P1xelC0da** [reply]
Got it, works all good now. Thanks for this amazing software.
